client_test/helpers_test.go

```diff
@@ -229,3 +231,22 @@ func closeSimulatedChain(t *testing.T, chain chainservice.SimulatedChain) {
 		t.Fatal(err)
 	}
 }
+
+// generateMessageKey generates a ECDSA private key deterministically using the given pk bytes.
+func generateMessageKey(pk []byte) p2pcrypto.PrivKey {
+
```

Comment:
```suggestion
```

Reply:
It looks like there's an open issue for [gofmt](https://github.com/golang/go/issues/6996) but that led me to this [tool](https://github.com/mvdan/gofumpt)